docs/portal/image-registry/deleting-projects.md

```diff
@@ -0,0 +1,5 @@
+# Deleting Projects
+
+If you no longer need a project, you can delete it from the image registry. To make sure you delete the appropriate project, you can view its information before deleting it.
```

Comment:
Deleting a project requires admin portal user permission as well. Please add a caveat on this page. Sorry, I missed this the other day.

Reply:
Fixed, thank you.